tools/package_tor.js

```diff
@@ -33,11 +33,11 @@ if (!fs.existsSync(torPath)) {
 
 var sha512Tor
 if (isDarwin) {
-  sha512Tor = '1a578a544ba259a9de11a63ef24f867bb7efbf7df4cd45dd08b9fff775f3b7f39eacd699c25fab22d69d4bee25bc03e9977a5cc66416792281276d584c101a5f'
+  sha512Tor = 'ecd26c84a95785475ff4a53852716b99df7c73b041b755973041cee295e91fe809862537ebb9cc093c83e56d4bc58b56843fddae357fbdd05eef34e9e3094f05'
 } else if (isLinux) {
-  sha512Tor = '193f01b75123debf90b3e35d0bc731f9e59cc06cd4e2869123f133f3a5f5c1796150b536c3cca50a9579c03f90084e5052d3ca385f807eac191f46348a57dce1'
+  sha512Tor = '18cd93d74929e89cb668442b1a405d1a90e9bef81bf0fb046234054b3a86e0aaa0eb4ffbbe739af2f4af342c04d7871458028f0609b3942d8e1c0211d4a5b899'
 } else {
-  sha512Tor = '7ba514fdd5f184015d65bbb65c82475dc256d23078dd3f7d115b4e2b93bf73ceedfbca89eed1baf501bddcdb7f5c81f384e02836588d067c01c3f469b0664885'
+  sha512Tor = '1a2247bebd8fb02a08c8bd6661ba1b98b179abd46f2b6ab930bc91c80c88b1b82dfa212ef0f89d8ccd6e971e29a7cdfce809929188b3bc7eddcd537583ba5c74'
```

Comment:
just to confirm, is this the mingw build? (not the old 0.3.2 build)

Reply:
yes